cluster-service/Makefile

```diff
@@ -18,14 +18,19 @@ deploy:
 	AZURE_CS_MI_CLIENT_ID=$(shell az identity show \
 			-g ${RESOURCEGROUP} \
 			-n clusters-service \
-			--query clientId) && \
+			--query clientId) \
+	AZURE_FIRST_PARTY_APPLICATION_CLIENT_ID=$(shell az ad app list \
+			--display-name aro-hcp-dev-first-party \
+			--query '[*]'.appId \
+			-o tsv) && \ 
```

Comment:
what does the `&&` do here? It was removed in the assignment of `AZURE_CS_MI_CLIENT_ID`

Reply:
this needs to be here and also another one in line 21 is required. the env vars have only scope in the same command they are used, which is the `oc process` in line 27

Reply:
ah interesting, i thought they didn't have to be in the same command to be in scope. Thanks for confirming!